src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/Clients/ContentApi/ContentApiClient.cs

```diff
@@ -62,162 +68,104 @@ public async Task<ReleaseSearchableDocument> GetPublicationLatestReleaseSearchab
         string publicationSlug,
         CancellationToken cancellationToken)
     {
-        var apiEndpoint = string.Format(GetPublicationLatestReleaseSearchViewModelApiEndpointFormat, publicationSlug);
+        var apiEndpoint = BuildGetPublicationLatestReleaseSearchViewModelApiEndpoint(publicationSlug);
         var response = await Get<ReleaseSearchViewModelDto>(apiEndpoint, cancellationToken);
 
-        return response switch
-        {
-            GetResponse<ReleaseSearchViewModelDto>.Success success =>
-                success.Result.ToModel(),
-
-            GetResponse<ReleaseSearchViewModelDto>.Error error =>
-                throw new UnableToGetPublicationLatestReleaseSearchViewModelException(
-                    publicationSlug,
-                    error.ErrorMessage),
-
-            _ => throw new UnreachableException()
-        };
+        return Process(
+            response,
+            onSuccess: dto => dto.ToModel(),
+            onError: errorMessage => new UnableToGetPublicationLatestReleaseSearchViewModelException(publicationSlug, errorMessage));
     }
 
-    public async Task<(bool WasSuccesssful, string? ErrorMessage)> Ping(CancellationToken cancellationToken)
+    public async Task<(bool WasSuccesssful, string? ErrorMessage)> Ping(CancellationToken cancellationToken = default)
     {
         // Determine whether the Content API is available by making a simple call
-        var apiEndpoint = BuildGetPublicationsPageEndpoint(page: 1, numberOfItems: 1);
-        var response = await Get<PaginatedResultDto<PublicationDto>>(apiEndpoint, cancellationToken);
+        var response = await Get<object>(GetThemesEndpoint, cancellationToken);
+        
         return response switch
         {
-            GetResponse<PaginatedResultDto<PublicationDto>>.Success => (true, null),
-            GetResponse<PaginatedResultDto<PublicationDto>>.Error error => (false, error.ErrorMessage),
+            GetResponse<object>.Success => (true, null),
+            GetResponse<object>.Error error => (false, error.ErrorMessage),
             _ => throw new UnreachableException()
         };
```

Comment:
Is it possible to use `Process` here like the rest of the methods, or is it because no exception gets thrown that we don't want to use it?